src/shardguard/core/prompts.py

```diff
@@ -11,15 +11,19 @@
 1. **Identify** any and all specific piece of sensitive or private data in the user prompt
     (medical conditions, health info, personal names, addresses, credentials,
     phone numbers, email details, etc.—anything a privacy-minded reviewer would mask).
-2. If they exist, then **replace** each unique value with a placeholder you invent, following the
+2. If they **exist**, then **replace** each unique value with a placeholder you invent, following the
```

Comment:
Why are you changing this file as part of adding the Nuke server?   This seems unrelated.